custom/ilsgateway/zipline/filters.py

```diff
@@ -0,0 +1,24 @@
+from corehq.apps.reports.filters.base import BaseMultipleOptionFilter
+from custom.zipline.models import EmergencyOrderStatusUpdate
+
+
+class EmergencyOrderStatusChoiceFilter(BaseMultipleOptionFilter):
+    slug = 'statuses'
+    label = 'Status'
+
+    @property
+    def options(self):
+        return [
+            (EmergencyOrderStatusUpdate.STATUS_PENDING, EmergencyOrderStatusUpdate.STATUS_PENDING),
+            (EmergencyOrderStatusUpdate.STATUS_ERROR, EmergencyOrderStatusUpdate.STATUS_ERROR),
+            (EmergencyOrderStatusUpdate.STATUS_RECEIVED, EmergencyOrderStatusUpdate.STATUS_RECEIVED),
+            (EmergencyOrderStatusUpdate.STATUS_REJECTED, EmergencyOrderStatusUpdate.STATUS_REJECTED),
+            (EmergencyOrderStatusUpdate.STATUS_APPROVED, EmergencyOrderStatusUpdate.STATUS_APPROVED),
+            (EmergencyOrderStatusUpdate.STATUS_CANCELLED, EmergencyOrderStatusUpdate.STATUS_CANCELLED),
+            (EmergencyOrderStatusUpdate.STATUS_DISPATCHED, EmergencyOrderStatusUpdate.STATUS_DISPATCHED),
+            (EmergencyOrderStatusUpdate.STATUS_DELIVERED, EmergencyOrderStatusUpdate.STATUS_DELIVERED),
+            (EmergencyOrderStatusUpdate.STATUS_CONFIRMED, EmergencyOrderStatusUpdate.STATUS_CONFIRMED),
+            (EmergencyOrderStatusUpdate.STATUS_CANCELLED_BY_USER,
+             EmergencyOrderStatusUpdate.STATUS_CANCELLED_BY_USER),
+
+        ]
```

Comment:
can we create a tuple of two-tuples in custom.zipline.models.EmergencyOrderStatusUpdate called STATUS_CHOICES, which has the status value as the first element in each two-tuple, and then a translatable name for the second value of each two-tuple which converts the camel-case style status into a more readable version. For example:

    STATUS_CHOICES = (
        (STATUS_PENDING, ugettext_lazy('Pending'),
        (STATUS_STATUS_CANCELLED_BY_USER, ugettext_lazy('Cancelled By User'),
        ...
    )

We can include all possible status values in STATUS_CHOICES, and then just use it here to look up the translated values: dict(STATUS_CHOICES)[EmergencyOrderStatusUpdate.STATUS_PENDING]